active/af1-identify-phi.py

```diff
@@ -274,19 +282,23 @@ def annotate_clinical_notes(**kwargs):
 
 def save_deid_annotations(annotation_records):
     for blobid, record in annotation_records.items():
-        common_functions.save_deid_annotation(blobid, record['hdcpupdatedate'], str(record['annotated_note']))
+        if record:
+            common_functions.save_deid_annotation(blobid, record['hdcpupdatedate'], str(record['annotated_note']))
 
 
 def save_unreviewed_annotations(annotation_records):
     for blobid, record in annotation_records.items():
-        common_functions.save_unreviewed_annotation(blobid, record['hdcpupdatedate'], str(record['annotated_note']))
+        if record:
+            common_functions.save_unreviewed_annotation(blobid, record['hdcpupdatedate'], str(record['annotated_note']))
 
```

Comment:
what's the else case here? there's no record because it wasn't retrieved (due to our good buddy adaptive)? or will we hit this if there are no annotations in the deid process?

Reply:
this is due to an edge case caused by a try/catch at line 235 (record isn't assigned because of an error earlier in the try/catch block)
I put some safeguards in the save_X fn's to guard against the case of key's without real values (empty dicts)